### PR TITLE
[OSDOCS#10565]: Add release notes for etcd tuning profiles

### PR DESCRIPTION
**Pull request by tmalove**
Version(s):
4.16

Issue:
[OSDOCS-10565](https://issues.redhat.com/browse/OSDOCS-10565)

Link to docs preview:
[Notable technical changes](https://75958--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-notable-technical-changes) (Updated 5/20/2024)

QE review:
- [ x] QE has approved this change.

### CODE REVIEW

**Comment by tmalove** [comment]
Hi @dusk125 @geliu2016 take a look at this PR for the release notes for etcd tuning profiles. Thanks!
cc: @kowen-rh

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done
/lgtm

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -240,6 +240,16 @@ For more information, see xref:../edge_computing/cnf-talm-for-cluster-upgrades.a
...
 
+[discrete]
+[id="ocp-4-16-etcd-tuning-profiles"]
+=== etcd tuning parameters
+
+With this release, the etcd tuning parameters can be set to values that optimize performance and decrease latency, as follows.
+
+* "" (Default)
+* Standard
+* Slower
```
Comment (michaelryanpeter) [inline release_notes/ocp-4-16-release-notes.adoc:251]: @tmalove: I think these parameters should be in backticks, which is what you had for the 4.15 TP docs: https://docs.openshift.com/container-platform/4.15/scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.html#etcd-tuning-parameters_recommended-etcd-practices

```suggestion
* `""` (Default)
* `Standard`
* `Slower`
```
Let me know what you think and feel free to DM me when you are ready for a merge.

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by michaelryanpeter** [comment]
Author requested a merge in a DM in Slack. Updates LGTM.

**Pull request merged**